src/main.py

```diff
@@ -26,6 +32,35 @@
 request = Request()
 
 
+@bot.event
+async def on_member_join(member: member_type) -> None:
+    LOGGER.warning("Member %s has just joined the server", member.name)
+
+    # Get welcome channel from server
+    channel: GuildChannel = bot.get_channel(int(os.getenv("WELCOME_CHANNEL_ID")))
```

Comment:
read the `WELCOME_CHANNEL_ID` from firestore, it is not a const var
It will change from one server to an other

Reply:
Changed, but not the idea solution from now